desktop/extensions-sdk/guides/invoke-host-binaries.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Invoke host binaries
+description: Add invocations to host binaries from the frontend with the extension SDK.
+keywords: Docker, extensions, sdk, build
+---
+
+In some cases, your extension needs to invoke some command from the host (the computer of your users). For example, you 
+might wand to invoke the CLI of your cloud provider to create a new resource, or the CLI of a tool your extension 
+provides. 
+You could do that executing the CLI from a container with the extension SDK. But this CLI needs to access the host's 
+filesystem, which isn't possible if it runs in a container.
+Host binaries allows exactly this: to invoke binaries from the host. As extensions can run on multiple platforms, 
+this means that you need to ship the binaries for all the platforms you want to support.
+
+In this example, this CLI will be a simple `Hello world` script that must be invoked with a parameter and will return a 
+string. Since the extension will support multiple platforms, the script will be written in `bash` for macOS and Linux, 
+and in `powershell` for Windows.
+
+## Add the binaries to the extension
```

Comment:
I think this section is confusing as it mentions binaries but the examples are about shell scripts. I was expecting an example with a real binary, like `kubectl`.

Reply:
That is interesting! You are right. But, since any script can be used here, be it binaries or shell scripts, should we rename host binaries as host as scripts instead? 
I think I can reuse `kubectl` here but IIRC I thought it would be repetitive with the guide about interacting with k8s.

Reply:
I discussed with @gtardif we decided to talk about executables rather than binaries in the doc.

Reply:
@benja-M-1 Executables seems to be a term more accurate. I hope we can say in the docs that we use the term binary and executable interchangeably (to not confuse users with the word `binaries` in `metadata.json`)